pontoon/sync/formats/ftl.py

```diff
@@ -42,6 +45,8 @@ def __init__(self, path, locale, source_resource=None):
         self.entities = {}
         self.source_resource = source_resource
         self.order = 0
+        self.parser = FluentParser()
+        self.serializer = FluentSerializer()
```

Comment:
It might be a good idea to instantiate the parser and the serializer once at the top of this file.